web/server/vue-cli/src/components/Product/ProductConfigForm.vue

```diff
@@ -169,6 +177,12 @@ export default {
   },
 
   computed: {
+    confidentiality_items() {
```

Comment:
In JavaScript we are using camel cased variable names:
```suggestion
        confidentialityItems() {
```